### PR TITLE
Remove references to SCA Part 2

### PR DESCRIPTION
**Pull request by bangelic**
Related to #2745 

This is for 6.16

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.10/Katello 4.12
* [ ] Foreman 3.9/Katello 4.11 (planned Satellite 6.15)
* [ ] Foreman 3.8/Katello 4.10
* [ ] Foreman 3.7/Katello 4.9 (Satellite 6.14)
* [ ] Foreman 3.6/Katello 4.8
* [ ] Foreman 3.5/Katello 4.7 (Satellite 6.13; orcharhino 6.6/6.7)
* [ ] Foreman 3.4/Katello 4.6 (EL8 only)
* [ ] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only; orcharhino 6.4/6.5 on EL8 only)
* We do not accept PRs for Foreman older than 3.3.

### CODE REVIEW

**Comment by bangelic** [comment]
@jeremylenz Review and let me have it! :-)

File: guides/common/modules/proc_creating-an-activation-key.adoc [hunk 1]
```diff
@@ -29,6 +28,8 @@ ifndef::orcharhino[]
 endif::[]
 . Click *Save*.
 . Optional: In the *System Purpose* section, you can configure the activation key with system purpose to set on hosts during registration.
+You can configure the activation key to set system purpose attributes on hosts during registration.
```
Comment (jeremylenz) [inline guides/common/modules/proc_creating-an-activation-key.adoc:31]: This says the same thing twice.

```suggestion
. Optional: In the *System Purpose* section, you can configure the activation key to set system purpose attributes on hosts during registration.
```

File: guides/common/modules/proc_creating-an-activation-key.adoc [hunk 2]
```diff
@@ -1,20 +1,19 @@
...
+If there are subscriptions specified and auto-attach is enabled, hosts using the activation key select the best fitting subscription from the list specified in the activation key.
+Setting system purpose on the activation key does not affect this scenario.
+. Activation key with the exact set of subscriptions.
+If there are subscriptions specified and auto-attach is disabled, hosts using the activation key are associated with all subscriptions specified in the activation key.
+Setting system purpose on the activation key does not affect this scenario.
 
-Note that {Project} automatically attaches subscriptions only for the products installed on a host.
-For subscriptions that do not list products installed on {RHEL} by default, such as the Extended Update Support (EUS) subscription, use an activation key specifying the required subscriptions.
+.Custom Products
+If a {customproduct}, typically containing content not provided by Red Hat, is assigned to an activation key, this product is always enabled for the registered content host regardless of the auto-attach setting.
```
Comment (jeremylenz) [inline guides/common/modules/proc_creating-an-activation-key.adoc:14]: Remove all of this (lines 4-16, from "Activation key attaches subscriptions automatically" to and including "If a {customproduct}..."

File: guides/common/modules/proc_creating-an-activation-key.adoc [hunk 3]
```diff
@@ -1,20 +1,19 @@
 [id="Creating_an_Activation_Key_{context}"]
 = Creating an activation key
```
Comment (jeremylenz) [inline guides/common/modules/proc_creating-an-activation-key.adoc:2]: As mentioned in https://github.com/theforeman/foreman-documentation/pull/2745#discussion_r1504337915 , should add a blurb here like

```
Create an activation key to assign various attributes to hosts at registration time.
```

File: guides/common/modules/proc_creating-an-activation-key.adoc [hunk 4]
```diff
@@ -81,6 +82,12 @@ endif::[]
 --organization "_My_Organization_"
 ----
 +
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# hammer activation-key product-content \
+--name "_My_Activation_Key_" \
+--organization "_My_Organization_"
+----
```
Comment (jeremylenz) [inline guides/common/modules/proc_creating-an-activation-key.adoc:90]: This doesn't seem to be needed. Doesn't it just repeat the command directly above it?

edit: Oh, I see the one above has `--content-access-mode-all true`. I think we do need that for SCA, so this change should be removed and we should just keep the existing one.

**Review by jeremylenz — COMMENTED** [review]
Please make sure all resolved comments from https://github.com/theforeman/foreman-documentation/pull/2745 are addressed here :)

Some more changes needed to files not in this PR:

https://theforeman-foreman-documentation-preview-pr-2909.surge.sh/nightly/Managing_Content/index-katello.html#Adding_Red_Hat_Subscriptions_to_Subscription_Allocations_content-management - Should rename "subscription allocation" to "subscription manifest" throughout all docs, since this seems to be the prevailing terminology nowadays. 

https://theforeman-foreman-documentation-preview-pr-2909.surge.sh/nightly/Managing_Content/index-katello.html#Removing_Red_Hat_Subscriptions_from_Subscription_Allocations_content-management - same here

https://theforeman-foreman-documentation-preview-pr-2909.surge.sh/nightly/Managing_Content/index-katello.html#Attaching_Red_Hat_Subscriptions_to_Content_Hosts_content-management - This entire section can be removed. Subscriptions are not attached to content hosts anymore.

https://theforeman-foreman-documentation-preview-pr-2909.surge.sh/nightly/Managing_Content/index-katello.html#Updating_Red_Hat_Subscriptions_on_Multiple_Hosts_content-management - This entire section can be removed.

https://theforeman-foreman-documentation-preview-pr-2909.surge.sh/nightly/Managing_Content/index-katello.html#Creating_an_Activation_Key_short_content-management - This section needs reworking similar to the ones already here in this PR.

https://theforeman-foreman-documentation-preview-pr-2909.surge.sh/nightly/Managing_Content/index-katello.html#Creating_an_Activation_Key_to_Consume_the_Content_View_content-management - same here

https://theforeman-foreman-documentation-preview-pr-2909.surge.sh/nightly/Managing_Content/index-katello.html#Managing_Activation_Keys_content-management - same here

https://theforeman-foreman-documentation-preview-pr-2909.surge.sh/nightly/Managing_Content/index-katello.html#Updating_Subscriptions_Associated_with_an_Activation_Key_content-management - This section can be removed

https://theforeman-foreman-documentation-preview-pr-2909.surge.sh/nightly/Managing_Content/index-katello.html#Using_Activation_Keys_for_Host_Registration_content-management - needs rework

https://theforeman-foreman-documentation-preview-pr-2909.surge.sh/nightly/Managing_Content/index-katello.html#Setting_the_Service_Level_content-management - opening blurb needs rework

https://theforeman-foreman-documentation-preview-pr-2909.surge.sh/nightly/Managing_Content/index-katello.html#Content_Delivery_Network_Structure_content-management - blurb needs rework


I found references to auto-attach here:

https://theforeman-foreman-documentation-preview-pr-2909.surge.sh/nightly/Managing_Content/index-katello.html#Creating_an_Activation_Key_content-management


In the Managing Hosts guide, I also found some more places that need rework:

https://theforeman-foreman-documentation-preview-pr-2909.surge.sh/nightly/Managing_Hosts/index-katello.html#host-substatus-overview_managing-hosts

https://theforeman-foreman-documentation-preview-pr-2909.surge.sh/nightly/Managing_Hosts/index-katello.html#overview-of-the-host-columns_managing-hosts - (references to subscription-related host statuses)

https://theforeman-foreman-documentation-preview-pr-2909.surge.sh/nightly/Managing_Hosts/index-katello.html#Using_Report_Templates_to_Monitor_Hosts_managing-hosts - "subscription details"

https://theforeman-foreman-documentation-preview-pr-2909.surge.sh/nightly/Managing_Hosts/index-katello.html#Generating_Host_Monitoring_Reports_managing-hosts - Days from Now input is now moved to the Subscription - General report. need to update these instructions

In the Load Balancer guide, there's a reference to "Attaching the Foreman Infrastructure Subscription": https://theforeman-foreman-documentation-preview-pr-2909.surge.sh/nightly/Configuring_Load_Balancer/index-katello.html#Prerequisites_for_Configuring_smart-proxy_Servers_for_Load_Balancing_load-balancing

In the Administering {Project} guide:

https://theforeman-foreman-documentation-preview-pr-2909.surge.sh/nightly/Administering_Project/index-katello.html#Monitoring_Resources_admin - needs "subscriptions" reference removed

https://theforeman-foreman-documentation-preview-pr-2909.surge.sh/nightly/Administering_Project/index-katello.html#Using_the_Project_Content_Dashboard_admin - references to subscription-related host statuses

**Comment by jeremylenz** [comment]
> This is for 6.16

Oh, and this can be cherry-picked to 4.12 as well.

File: guides/common/modules/con_host-substatus-overview.adoc [hunk 5]
```diff
@@ -169,76 +169,6 @@ Possible values:
 | SCA enabled | OK | 5
 |===
 
-Service level::
```
Comment (jeremylenz) [inline guides/common/modules/con_host-substatus-overview.adoc:172]: Subscription status is also removed. Need to remove the section above and all other mentions of subscription status.

File: guides/common/modules/proc_creating-an-activation-key-to-consume-the-content-view.adoc [hunk 6]
```diff
@@ -1,7 +1,7 @@
 [id="Creating_an_Activation_Key_to_Consume_the_Content_View_{context}"]
 = Creating an activation key to consume the content view
 
-Use the following procedure to create an activation key that you can use to subscribe your hosts to content managed by {Project}.
+Create an activation key to assign various attributes to content managed by {Project}.
```
Comment (jeremylenz) [inline guides/common/modules/proc_creating-an-activation-key-to-consume-the-content-view.adoc:4]:
```suggestion
Create an activation key to assign various attributes to hosts at registration time.
```

The attributes are assigned to hosts, not to content.

File: guides/common/modules/proc_creating-an-activation-key.adoc [hunk 7]
```diff
@@ -1,20 +1,9 @@
 [id="Creating_an_Activation_Key_{context}"]
 = Creating an activation key
 
-Subscription Manager attaches subscriptions differently depending on the following factors:
+Create an activation key to assign various attributes to hosts during registration.
```
Comment (jeremylenz) [inline guides/common/modules/proc_creating-an-activation-key.adoc:4]: This wording works for me too :+1:

File: guides/common/modules/proc_creating-an-activation-key.adoc [hunk 8]
```diff
@@ -44,18 +34,6 @@ endif::[]
...
-[options="nowrap", subs="+quotes,verbatim,attributes"]
-----
-# hammer activation-key update \
---organization "_My_Organization_" \
---name "_My_Activation_Key_" \
---service-level "_Standard_" \
---purpose-usage "_Development/Test_" \
---purpose-role "_{RHELServer}_" \
---purpose-addons "_addons_"
-----
```
Comment (jeremylenz) [inline guides/common/modules/proc_creating-an-activation-key.adoc:58]: This should be left in. The only part that's no longer accurate is "to enhance subscriptions auto attachment."

Also, system purpose is not limited to RHEL8.

Perhaps
```
Optional: Enter the following command to configure the activation key with system purpose attributes to set on hosts during registration.
This can help determine which repositories are available on the host, and also helps with reporting in the Subscriptions service of the Red Hat Hybrid Cloud Console.
```

File: guides/common/modules/proc_generating-host-monitoring-reports.adoc [hunk 9]
```diff
@@ -36,22 +37,22 @@ If you want to generate the report as a background task, you can use the `hammer
 +
 [NOTE]
 ====
-If you want to generate a subscription entitlement report, you have to use the `Days from Now` option to specify the latest expiration time of entitlement subscriptions.
-You can use the `no limit` value to show all entitlements.
+If you want to generate a subscription general report, you have to use the `Days from Now` option to specify the latest expiration time of general subscriptions.
```
Comment (jeremylenz) [inline guides/common/modules/proc_generating-host-monitoring-reports.adoc:40]: This is referring to a specific report.

```suggestion
If you want to generate the Subscription - General report, you have to use the `Days from Now` option to specify the latest expiration time of general subscriptions.
```

File: guides/common/modules/proc_generating-host-monitoring-reports.adoc [hunk 10]
```diff
@@ -36,22 +37,22 @@ If you want to generate the report as a background task, you can use the `hammer
...
+You can use the `no limit` value to show all inputs.
```
Comment (jeremylenz) [inline guides/common/modules/proc_generating-host-monitoring-reports.adoc:41]: Subscriptions, not inputs.  (Yes, subscriptions are still around in your manifest, and that's what this report shows :)
```suggestion
You can use the `no limit` value to show all subscriptions.
```

File: guides/common/modules/proc_generating-host-monitoring-reports.adoc [hunk 11]
```diff
@@ -36,22 +37,22 @@ If you want to generate the report as a background task, you can use the `hammer
...
 
-.Show all entitlements
+.Show all inputs
```
Comment (jeremylenz) [inline guides/common/modules/proc_generating-host-monitoring-reports.adoc:43]:
```suggestion
.Show all subscriptions
```

File: guides/common/modules/proc_generating-host-monitoring-reports.adoc [hunk 12]
```diff
@@ -36,22 +37,22 @@ If you want to generate the report as a background task, you can use the `hammer
...
 [options="nowrap", subs="+quotes,attributes,verbatim"]
 ----
 # hammer report-template generate \
 --inputs "Days from Now=no limit" \
---name "Subscription - Entitlement Report"
+--name "Subscription - General Report"
 ----
 
-.Show all entitlements that are going to expire within 60 days
+.Show all inputs that are going to expire within 60 days
```
Comment (jeremylenz) [inline guides/common/modules/proc_generating-host-monitoring-reports.adoc:51]:
```suggestion
.Show all subscriptions that are going to expire within 60 days
```

File: guides/common/modules/proc_setting-the-service-level.adoc [hunk 13]
```diff
@@ -1,9 +1,7 @@
 [id="Setting_the_Service_Level_{context}"]
 = Setting the service level
 
-You can configure an activation key to define a default service level for the new host created with the activation key.
-Setting a default service level selects only the matching subscriptions to be attached to the host.
-For example, if the default service level on an activation key is set to Premium, only subscriptions with premium service levels are attached to the host upon registration.
+You can configure an activation key to define a default service level for the new host assigned various attributes during registration.
```
Comment (jeremylenz) [inline guides/common/modules/proc_setting-the-service-level.adoc:4]: Even though subscription attachments are no longer involved, service level (and other syspurpose attributes) still affect availability of repositories on hosts.

```suggestion
You can configure an activation key to define a default service level for the new host created with the activation key.
Setting a default service level makes specific repositories available to the host.
For example, if the default service level on an activation key is set to Premium, only repositories with premium service levels are available to the host upon registration.
```

File: guides/common/modules/proc_using-activation-keys-for-host-registration.adoc [hunk 14]
```diff
@@ -6,21 +6,11 @@ You can use activation keys to complete the following tasks:
...
 You can use multiple activation keys when registering a content host.
-You can then create activation keys for specific subscription sets and combine them according to content host requirements.
-For example, the following command registers a content host to your organization with both VDC and OpenShift subscriptions:
-
-[options="nowrap" subs="+quotes"]
-----
-# subscription-manager register \
---activationkey="ak-VDC,ak-OpenShift" \
---org="_My_Organization_"
-----
```
Comment (jeremylenz) [inline guides/common/modules/proc_using-activation-keys-for-host-registration.adoc:23]: You can still use multiple activation keys. We could replace this with another example, such as

```
For example, you could use one activation key to enable specific repositories, and another to assign a content view and lifecycle environment.
```

Since global registration is now the supported registration method and `subscription-manager register` is no longer recommended, I think we can leave off the command.

File: guides/common/modules/proc_using-activation-keys-for-host-registration.adoc [hunk 15]
```diff
@@ -6,21 +6,11 @@ You can use activation keys to complete the following tasks:
...
 
 .Settings conflicts
 If there are conflicting settings in activation keys, the rightmost key takes precedence.
```
Comment (jeremylenz) [inline guides/common/modules/proc_using-activation-keys-for-host-registration.adoc:16]: Down below, remove "Subscriptions" from

```
* Settings that do not conflict and the host gets the union of them: *Subscriptions* and *Host Collections*.
```

**Review by jeremylenz — COMMENTED** [review]
Looking good!

Once these are addressed I'll look thru the previews again.

File: guides/common/modules/proc_creating-an-activation-key-to-consume-the-content-view.adoc [hunk 16]
```diff
@@ -1,7 +1,7 @@
 [id="Creating_an_Activation_Key_to_Consume_the_Content_View_{context}"]
 = Creating an activation key to consume the content view
 
-Use the following procedure to create an activation key that you can use to subscribe your hosts to content managed by {Project}.
+Create an activation key to assign various attributes to hosts during registration.
```
Comment (jeremylenz) [inline guides/common/modules/proc_creating-an-activation-key-to-consume-the-content-view.adoc:4]: This procedure seems to be specific to this workflow, and not general. I think we could actually just leave this sentence out.

Below, need to remove

```
. Click the *Subscriptions* tab on the Activation Key's details page.
. Click the *Add* tab.
. Select your new *CentOS Stream* product in the table.
. Click *Add Selected* to save.
```

File: guides/common/modules/proc_creating-an-activation-key.adoc [hunk 17]
```diff
@@ -28,7 +17,8 @@ ifndef::orcharhino[]
 . In the *Repository Sets* tab, ensure only your named repository is enabled.
```
Comment (jeremylenz) [inline guides/common/modules/proc_creating-an-activation-key.adoc:17]: I feel like this was left over from when custom products were enabled by default. In any case, I think this step could be removed. Or, we could just say more generally

```
. In the *Repository Sets* tab, override repositories to Enabled or Disabled as desired. For more information, see xxx
```

File: guides/common/modules/proc_creating-an-activation-key.adoc [hunk 18]
```diff
@@ -44,7 +34,8 @@ endif::[]
 --content-view "_Stack_" \
 --organization "_My_Organization_"
 ----
-. Optional: For {RHEL} 8 hosts, enter the following command to configure the activation key with system purpose to set on hosts during registration to enhance subscriptions auto attachment.
+. Optional: Enter the following command to configure the activation key with system purpose attributes to set on hosts during registration.
+This helps determine which repositories are available on the host as well as with reporting in the Subscriptions service of the Red Hat Hybrid Cloud Console.
 +
```
Comment (jeremylenz) [inline guides/common/modules/proc_creating-an-activation-key.adoc:39]: Need to remove some stuff below this

```
. Obtain a list of your subscription IDs:
+
[options="nowrap", subs="+quotes,verbatim,attributes"]
----
# hammer subscription list --organization "_My_Organization_"
----
. Attach the {RHEL} subscription UUID to the activation key:
+
[options="nowrap", subs="+quotes,verbatim,attributes"]
----
# hammer activation-key add-subscription \
--name "_My_Activation_Key_" \
--name "_My_Activation_Key_" \
--organization "_My_Organization_"
----
```

File: guides/common/modules/proc_setting-the-service-level.adoc [hunk 19]
```diff
@@ -1,9 +1,9 @@
 [id="Setting_the_Service_Level_{context}"]
 = Setting the service level
 
-You can configure an activation key to define a default service level for the new host created with the activation key.
-Setting a default service level selects only the matching subscriptions to be attached to the host.
-For example, if the default service level on an activation key is set to Premium, only subscriptions with premium service levels are attached to the host upon registration.
+You can configure an activation key to define a default service level for the new host created with it.
+Setting a default service level makes specific repositories available to the host.
+For example, if the default service level on an activation key is set to Premium, only repositories with premium service levels are available to the host after registration.
```
Comment (jeremylenz) [inline guides/common/modules/proc_setting-the-service-level.adoc:6]: I'm pretty sure repositories without a service level set will also be available. (It's only repos with non-matching service levels that would be hidden.)

```suggestion
For example, if the default service level on an activation key is set to Premium, repositories with premium service levels are available to the host after registration.
```

**Review by jeremylenz — COMMENTED** [review]
A few more comments below.

Looking at the previews now, stand by for more ;)

File: guides/common/modules/proc_creating-an-activation-key-to-consume-the-content-view.adoc (hunk 16 above)
Comment (bangelic) [reply guides/common/modules/proc_creating-an-activation-key-to-consume-the-content-view.adoc:4]: @jeremylenz Do you have any suggestions for a new opening sentence to replace `Create an activation key to assign various attributes to hosts during registration.` ?

**Review by jeremylenz — COMMENTED** [review]
### We can leave these for a follow-up, since they relate to the base system Satellite is installed on and not Satellite itself

-----------------------------------------------------

I think the subscription-related steps here (1 and 2) may not be needed if your Satellite server itself is on a Red Hat account that uses SCA. Not sure how that should be handled. @ehelms @evgeni thoughts?

https://theforeman-foreman-documentation-preview-pr-2909.surge.sh/nightly/Administering_Project/index-satellite.html#sec-Preparing_Source_Server

Same with Step 4 here:
https://theforeman-foreman-documentation-preview-pr-2909.surge.sh/nightly/Administering_Project/index-satellite.html#sec-Cloning_to_Target

Same with https://theforeman-foreman-documentation-preview-pr-2909.surge.sh/nightly/Administering_Project/index-satellite.html#preparing-a-host-for-external-databases_admin
^ This one seems to have a reference to "Attaching the Satellite Infrastructure Subscription" which we removed the reference to in this PR.

One of the steps there says "If you use the virt-who agent, update the virt-who configuration files with the new host name. For more information, see [Modifying a virt-who Configuration](https://access.redhat.com/documentation/en-us/red_hat_satellite/6.15/html-single/configuring_virtual_machine_subscriptions/index#troubleshooting-virt-who#modifying-virt-who-configuration_vm-subs-satellite) in Configuring virtual machine subscriptions." I can't figure out where that link is supposed to lead, but it probably needs some rework as well.

### These ones below should be addressed now

---------------------------------------------


- [x] In https://theforeman-foreman-documentation-preview-pr-2909.surge.sh/nightly/Managing_Content/index-katello.html#Content_Delivery_Network_Structure_content-management - 

```
The accessible subset of the CDN is configured through subscriptions attached to a system using [Red Hat Subscription Management](https://access.redhat.com/management/) or using Foreman server.
```
should change "subscriptions attached to a system" to "content available to a system".

- [x] https://theforeman-foreman-documentation-preview-pr-2909.surge.sh/nightly/Managing_Content/index-katello.html#Managing_Activation_Keys_content-management

This needs a rework, maybe this one was missed before? we can talk offline about it if you want

- [x] https://theforeman-foreman-documentation-preview-pr-2909.surge.sh/nightly/Managing_Hosts/index-satellite.html#Creating_a_Host_managing-hosts
Step 7 (the Parameters step) needs rework re: System Purpose

- [ ] https://theforeman-foreman-documentation-preview-pr-2909.surge.sh/nightly/Managing_Hosts/index-satellite.html#Editing_the_System_Purpose_of_a_Host_managing-hosts
need to remove Step 3 about auto-attach

- [ ] https://theforeman-foreman-documentation-preview-pr-2909.surge.sh/nightly/Managing_Hosts/index-satellite.html#Editing_the_System_Purpose_of_Multiple_Hosts_managing-hosts
Same kind of edits here, for multiple.

File: guides/common/modules/proc_creating-an-activation-key-to-consume-the-content-view.adoc (hunk 16 above)
Comment (jeremylenz) [reply guides/common/modules/proc_creating-an-activation-key-to-consume-the-content-view.adoc:4]: The procedure is guiding you through creating an AK so that your host consumes the content view, so I couldn't really think of anything to replace it that didn't just re-state the title. Do we have to have an opening sentence?

This entire chapter ("Basic Content Management Workflow") seems designed so that you follow all the procedures in order, rather than just picking out one. 

I guess we could say something like

```
Create another activation key to assign hosts to the content view you created in the previous procedure.
```

Comment (bangelic) [reply guides/common/modules/proc_creating-an-activation-key-to-consume-the-content-view.adoc:4]: Yeah we need an intro of some kind. I think what you said here works perfectly.

File: guides/common/modules/proc_creating-an-activation-key.adoc [hunk 20]
```diff
@@ -14,7 +14,8 @@ If you want unlimited hosts to register with the activation key, ensure the *Unl
 . From the *Environment* list, select the environment to use.
 . From the *Content View* list, select a content view to use.
 ifndef::orcharhino[]
-. In the *Repository Sets* tab, ensure only your named repository is enabled.
+. In the *Repository Sets* tab, override repositories to Enabled or Disabled as desired.
+For more information, see {ContentManagementDocURL}Enabling_Red_Hat_Repositories_content-management[Enabling Red Hat repositories] in _{ContentManagementDocTitle}_.
```
Comment (jeremylenz) [inline guides/common/modules/proc_creating-an-activation-key.adoc:18]: Wrong kind of repository enabling (I know..)

This should instead link to https://theforeman-foreman-documentation-preview-pr-2909.surge.sh/nightly/Managing_Content/index-katello.html#enabling-and-disabling-repositories-on-activation-key_content-management

File: guides/common/modules/proc_editing-the-system-purpose-of-multiple-hosts.adoc [hunk 21]
```diff
@@ -2,14 +2,10 @@
 = Editing the system purpose of multiple hosts
 
 You can edit the system purpose attributes of {RHEL} hosts.
-System purpose attributes define which subscriptions to attach automatically to hosts.
 ifndef::orcharhino[]
 For more information about system purpose, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/performing_a_standard_rhel_8_installation/graphical-installation_graphical-installation?extIdCarryOver=true&sc_cid=701f2000001Css5AAC#system_purposes-adoc_connect-to-red-hat[Introduction to System Purpose] in _Performing a standard RHEL 8 installation_.
 endif::[]
 
-.Prerequisites
-* The hosts that you want to edit must be registered with the subscription-manager.
```
Comment (jeremylenz) [inline guides/common/modules/proc_editing-the-system-purpose-of-multiple-hosts.adoc:11]: This should be left in.

File: guides/common/modules/proc_creating-a-host.adoc [hunk 22]
```diff
@@ -57,7 +57,7 @@ To use a parameter variable with an Ansible job template, you must add a *Host P
 ifdef::satellite[]
 +
 When you create a {RHEL} 8 host, you can set system purpose attributes.
```
Comment (jeremylenz) [inline guides/common/modules/proc_creating-a-host.adoc:59]: While we're here, this isn't restricted to RHEL8 either.

File: guides/common/modules/proc_editing-the-system-purpose-of-a-host.adoc [hunk 23]
```diff
@@ -36,12 +36,6 @@ endif::[]
...
 ----
-. Automatically attach subscriptions to this host:
-+
-[subs="+quotes"]
-----
-# subscription-manager attach --auto
-----
 . Verify the system purpose status for this host:
 +
 [subs="+quotes"]
```
Comment (jeremylenz) [inline guides/common/modules/proc_editing-the-system-purpose-of-a-host.adoc:41]: I just realized this step isn't needed anymore either.

**Review by jeremylenz — COMMENTED** [review]
Looking good! Just one more change I found.

It says it can't generate the preview now, but if it regenerates I'll check it one final time.

**Review by jeremylenz — COMMENTED** [review]
Ok, found a couple more:

https://theforeman-foreman-documentation-preview-pr-2909.surge.sh/nightly/Managing_Content/index-satellite.html#Content-Views-in-Satellite_content-management

```
Default Organization View
A Default Organization View is an application-controlled content view for all content that is synchronized to Satellite. With the Default Organization View, you can register a host to Satellite and access content by using a subscription and without manipulating content views and lifecycle environments.
```

Delete "by using a subscription".

https://theforeman-foreman-documentation-preview-pr-2909.surge.sh/nightly/Managing_Hosts/index-satellite.html#Creating_a_Report_Template_to_Monitor_Entitlements_managing-hosts

Delete this entire section.

I _think_ that should be it.

File: guides/common/modules/proc_creating-a-host.adoc [hunk 24]
```diff
@@ -56,8 +56,8 @@ This includes all Puppet Class, Ansible playbook parameters and host parameters
 To use a parameter variable with an Ansible job template, you must add a *Host Parameter*.
 ifdef::satellite[]
 +
-When you create a {RHEL} 8 host, you can set system purpose attributes.
-System purpose attributes define what subscriptions to attach automatically on host creation.
+When you create a host, you can set system purpose attributes.
+System purpose attributes help determine which repositories are available on the host as well as with reporting in the Subscriptions service of the Red Hat Hybrid Cloud Console.
```
Comment (asteflova) [inline guides/common/modules/proc_creating-a-host.adoc:60]:
```suggestion
System purpose attributes help determine which repositories are available on the host.
System purpose attributes also help with reporting in the Subscriptions service of the Red Hat Hybrid Cloud Console.
```
Does this make sense? I had to re-read the long sentence a few times to parse it :) Splitting it like this or in a similar way would help.

File: guides/common/modules/proc_creating-an-activation-key-to-consume-the-content-view.adoc [hunk 25]
```diff
@@ -1,7 +1,7 @@
 [id="Creating_an_Activation_Key_to_Consume_the_Content_View_{context}"]
 = Creating an activation key to consume the content view
 
-Use the following procedure to create an activation key that you can use to subscribe your hosts to content managed by {Project}.
+Create another activation key to assign hosts to the content view you created in {ContentManagementDocURL}Creating_a_Content_View__content-management[Creating a content view] in _{ContentManagementDocTitle}_.
```
Comment (asteflova) [inline guides/common/modules/proc_creating-an-activation-key-to-consume-the-content-view.adoc:4]:
```suggestion
Create another activation key to assign hosts to the content view you created in {ContentManagementDocURL}Creating_a_Content_View_content-management[Creating a content view] in _{ContentManagementDocTitle}_. 
```
This _might_ even fix the linkchecker failure.

Comment (asteflova) [reply guides/common/modules/proc_creating-an-activation-key-to-consume-the-content-view.adoc:4]: Wait. But shouldn't this be an xref? This module is in _Managing content_, isn't it? (It's hard to check without a preview.)

File: guides/common/modules/proc_creating-an-activation-key.adoc [hunk 26]
```diff
@@ -25,10 +14,12 @@ If you want unlimited hosts to register with the activation key, ensure the *Unl
 . From the *Environment* list, select the environment to use.
 . From the *Content View* list, select a content view to use.
 ifndef::orcharhino[]
-. In the *Repository Sets* tab, ensure only your named repository is enabled.
+. In the *Repository Sets* tab, override repositories to Enabled or Disabled as desired.
+For more information, see {ContentManagementDocURL}enabling-and-disabling-repositories-on-activation-key_content-management[Enabling and disabling repositories on activation key] in _{ContentManagementDocTitle}_.
```
Comment (asteflova) [inline guides/common/modules/proc_creating-an-activation-key.adoc:18]: I this this should be an xref too.

File: guides/common/modules/proc_creating-an-activation-key.adoc [hunk 27]
```diff
@@ -25,10 +14,12 @@ If you want unlimited hosts to register with the activation key, ensure the *Unl
...
 endif::[]
 . Click *Save*.
-. Optional: In the *System Purpose* section, you can configure the activation key with system purpose to set on hosts during registration.
+. Optional: In the *System Purpose* section, you can configure the activation key to set system purpose attributes on hosts during registration.
+This helps determine which repositories are available on the host as well as with reporting in the Subscriptions service of the Red Hat Hybrid Cloud Console.
```
Comment (asteflova) [inline guides/common/modules/proc_creating-an-activation-key.adoc:22]:
```suggestion
This helps determine which repositories are available on the host.
It also helps with reporting in the Subscriptions service of the Red Hat Hybrid Cloud Console.
```
Or another way of splitting the long sentence.

File: guides/common/modules/proc_creating-an-activation-key.adoc [hunk 28]
```diff
@@ -44,7 +35,8 @@ endif::[]
 --content-view "_Stack_" \
 --organization "_My_Organization_"
 ----
-. Optional: For {RHEL} 8 hosts, enter the following command to configure the activation key with system purpose to set on hosts during registration to enhance subscriptions auto attachment.
+. Optional: Enter the following command to configure the activation key with system purpose attributes to set on hosts during registration.
+This helps determine which repositories are available on the host as well as with reporting in the Subscriptions service of the Red Hat Hybrid Cloud Console.
```
Comment (asteflova) [inline guides/common/modules/proc_creating-an-activation-key.adoc:39]:
```suggestion
This helps determine which repositories are available on the host.
It also helps with reporting in the Subscriptions service of the Red Hat Hybrid Cloud Console.
```

File: guides/common/modules/proc_generating-host-monitoring-reports.adoc [hunk 29]
```diff
@@ -8,6 +8,7 @@ To schedule reports, configure a cron job or use the {ProjectWebUI}.
 . In the {ProjectWebUI}, navigate to *Monitor* > *Reports* > *Report Templates*.
 For example, the following templates are available:
 Host {endash} Installed Products:: This generates a report with installed product information along with other metrics, including system purpose attributes.
```
Comment (asteflova) [inline guides/common/modules/proc_generating-host-monitoring-reports.adoc:10]:
```suggestion
For example, the *Host {endash} Installed Products* template generates a report with installed product information along with other metrics, including system purpose attributes.
```
This is no longer a list, only one item is left now.

File: guides/common/modules/proc_generating-host-monitoring-reports.adoc [hunk 30]
```diff
@@ -36,22 +37,22 @@ If you want to generate the report as a background task, you can use the `hammer
 +
 [NOTE]
 ====
-If you want to generate a subscription entitlement report, you have to use the `Days from Now` option to specify the latest expiration time of entitlement subscriptions.
-You can use the `no limit` value to show all entitlements.
+If you want to generate the Subscription - General report, you have to use the `Days from Now` option to specify the latest expiration time of general subscriptions.
```
Comment (asteflova) [inline guides/common/modules/proc_generating-host-monitoring-reports.adoc:40]:
```suggestion
If you want to generate the *Subscription - General* report, you have to use the `Days from Now` option to specify the latest expiration time of general subscriptions.
```

File: guides/common/modules/proc_setting-the-service-level.adoc [hunk 31]
```diff
@@ -1,9 +1,9 @@
 [id="Setting_the_Service_Level_{context}"]
 = Setting the service level
 
-You can configure an activation key to define a default service level for the new host created with the activation key.
-Setting a default service level selects only the matching subscriptions to be attached to the host.
-For example, if the default service level on an activation key is set to Premium, only subscriptions with premium service levels are attached to the host upon registration.
+You can configure an activation key to define a default service level for the new host created with it.
```
Comment (asteflova) [inline guides/common/modules/proc_setting-the-service-level.adoc:4]:
```suggestion
You can configure an activation key to define a default service level for the new host created with the activation key.
```
I recommend to revert this change. Otherwise, it might not clear from a quick read what "it" refers to (activation key? or service level? or host?).

**Review by asteflova — CHANGES_REQUESTED** [review]
I reviewed the diff only and also ignored all wording that was already present in the previous version.

File: guides/common/assembly_using-report-templates.adoc [hunk 32]
```diff
@@ -15,7 +15,4 @@ include::modules/proc_importing-report-templates-using-the-api.adoc[leveloffset=
 include::modules/proc_generating-a-list-of-installed-packages.adoc[leveloffset=+1]
 
 ifdef::satellite[]
-include::modules/proc_creating-a-report-template-to-monitor-entitlements.adoc[leveloffset=+1]
-endif::[]
```
Comment (asteflova) [inline guides/common/assembly_using-report-templates.adoc:19]: @bangelic Could this be the reason why the build started failing? Can you try re-adding the endif?

File: guides/common/modules/proc_creating-an-activation-key-to-consume-the-content-view.adoc (hunk 25 above)
Comment (bangelic) [reply guides/common/modules/proc_creating-an-activation-key-to-consume-the-content-view.adoc:4]: You are correct. I was going back and forth between Hosts and Content that I must have mixed them up.

File: guides/common/modules/proc_creating-an-activation-key.adoc (hunk 27 above)
Comment (bangelic) [reply guides/common/modules/proc_creating-an-activation-key.adoc:22]: I don't see this anymore in my text editor.

File: guides/common/modules/proc_creating-an-activation-key.adoc (hunk 28 above)
Comment (bangelic) [reply guides/common/modules/proc_creating-an-activation-key.adoc:39]: I don't see this anymore in my text editor either.

**Comment by bangelic** [comment]
Merged in https://github.com/theforeman/foreman-documentation/pull/2945/commits after some strange build failures.
